.gitignore

```diff
@@ -44,3 +44,5 @@ npm-debug.log
 
 # Local environment configuration
 /environment.local
+
+/smoketest-*.tmp
```

Comment:
Why is this necessary since you have the `with tempfile.TemporaryDirectory...`

Reply:
I added `dir=` to that, placing the temp dir into the build directory. This makes the tempdir easier to find for post-mortems but I forgot to actually make clean-up optional by going back to  

```
cleanup=True
…
try:
    …
finally:
    if cleanup:
        rmdir(…)
```

Let me push that.